tvtk/pyface/ui/qt4/decorated_scene.py

```diff
@@ -158,6 +158,7 @@ def _save_snapshot(self):
                             "Geomview", "VRML", "Wavefront", "Open Inventor",
                             "Povray", "X3D"]
 
+            wildcard = ""
```

Comment:
And yes, I absolutely think that testing UI interactions is a good use of time: it's an essential for a professionally developed application that involves those UI interactions. There are some tools in Traits and PyFace that make testing of UI interactions somewhat easier.